pkg/errors/errors.go

```diff
@@ -26,6 +28,12 @@ func New(message string) error {
 	return pkgerrors.New(message)
 }
 
+// NewWithoutStack is like new but does NOT wrap with a stack
+// This is useful for exported errors
+func NewWithoutStack(message string) error {
+	return stderrors.New(message)
```

Comment:
since this will necessarily be for making errors at the bottom of the stack that should not themselves contain a stack, this should be appropriate (a plain error with the message and nothing else).